docs/source/events.rst

```diff
@@ -11,143 +11,248 @@ versions of Endless OS which are not recorded by current versions.
+
+See `T29247 <https://phabricator.endlessm.com/T29247>`_.
+
+:UUID name: ``EOS_UPDATER_METRIC_FAILURE`` in eos-updater
+
+.. versionadded:: 2.6.0
+
+Parental Controls Enabled
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any parental controls are enabled for the current user.
+
+Recorded every time the App Center checks for OS updates (using its
+``eos-updater plugin``). This was chosen as a convenient regular event
+which happens inside the user session, rather than because of some deeper
+link to parental controls.
+
+No identifying details about the user or which parental controls are
+enabled. It’s intended to allow aggregate statistics about how widely
+parental controls are enabled (in any form).
+
+See `T28741 <https://phabricator.endlessm.com/T28741#810046>`_.
+
+:UUID name: ``GS_PARENTAL_CONTROLS_USAGE_EVENT`` in gnome-software
+
+.. versionadded:: 3.8.0
+
+Parental Controls Changed
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Parental control app is closed.
+
+Recorded whenever ``malcontent-control`` is closed, presumably after the
+user has edited one or more parental controls.
+
+The payload contains the current parental controls settings for one user on
+the system, including identification about whether the user is an
+admin. Other user details (such as username or full name) are not
+included. The event is submitted multiple times, once for each (non-system)
+user on the system.
+
+The same event is also recorded at the end of Initial Setup if (and only
+if) parental controls were enabled for the main user during Initial
+Setup. In that case, the main user account has been set up as a child user,
+with no administrator privileges, and with parental controls enabled. A
+second user has been created as the administrator. The event contains the
+values of the initial parental controls settings, but no identifying
+information about the user (such as their username or full name). The
+intention is to allow comparisons of which parental controls are enabled
+initially by users, and which are enabled long term.
+
+See `T28741 <https://phabricator.endlessm.com/T28741#810046>`_ and `#101
+<https://github.com/endlessm/azafea/pull/101#discussion_r402610338>`_.
+
+The fields in the payload have the same semantics as the properties in the
+`AppFilter
+<https://gitlab.freedesktop.org/pwithnall/malcontent/-/blob/master/accounts-service/com.endlessm.ParentalControls.AppFilter.xml>`_
+interface.
+
+:UUID name: ``MCT_PARENTAL_CONTROLS_EVENT`` in malcontent and gnome-initial-setup
+
+.. versionadded:: 3.8.0
+
+Windows App Opened
+~~~~~~~~~~~~~~~~~~
+
+A user tries to open a ``.exe`` or ``.msi`` file.
+
+:UUID name: ``WINDOWS_APP_OPENED`` in eos-gates
+
+.. versionadded:: 2.1.5
+
+Time Spent In Session
+~~~~~~~~~~~~~~~~~~~~~
+
+Time spent in session by all computer users for given .
+
+.. versionadded:: 4.0.0
+
+Time Foreground App Is Opened
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Duration of an app open during last day.
```

Comment:
```suggestion
Time in seconds (?? minutes ??) that the given app was open in the foreground of an unlocked session within the given aggregation period.
```

Reply:
I’ll keep seconds, it’s probably useful for some apps that are open for a short amount of time. I don’t think that it’s particularly bad for privacy.